src/Dunet.csproj

```diff
@@ -49,7 +50,7 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.2.0" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.1.0" />
```

Comment:
I had to downgrade this to 4.1.0, because otherwise I was hitting https://github.com/dotnet/roslyn/issues/61333